src/manifest.json

```diff
@@ -5,7 +5,7 @@
   "background_color": "#2196f3",
   "display": "standalone",
   "Scope": "/",
-  "start_url": "/ovfplayer/index.html",
+  "start_url": "/teststaging/index.html",
```

Comment:
This is in a static file.  We need to change the context in the URL depending on which environment we are being deployed in.
I haven't found a nice way of doing this yet....suggestions welcome :-D

Reply:
oooh, how about something like this?
http://technowhisp.com/2018/05/05/pwa/dynamic-pwa-manifest/

Reply:
Maybe I can just change it to index.html
https://developer.mozilla.org/en-US/docs/Web/Manifest#start_url

Reply:
That seems to work!